pkg/expr/func.go

```diff
@@ -126,13 +124,17 @@ func (f *eqFunc) call(args0 interface{}, args1 interface{}) bool {
 		}
 		return matchWithWildcards(s0, s1)
 	case []byte:
-		if len(args0.([]byte)) == net.IPv4len || len(args0.([]byte)) == net.IPv6len {
+		return bytes.Equal(s0, args1.([]byte))
+	case net.IP:
```

Comment:
this doesn't seem right. this assumes that things are getting passed in as net.IPs, when they are actually getting passed in []byte. This PR will break existing use cases -- where our expressions pass in []bytes and not IP address.

we need this case to be []byte.